### PR TITLE
Add getraworphanpool rpc

### PR DESCRIPTION
**Pull request by ptschip**
This returns all the transaction hashes for transactions currently in
the orphanpool.

### CODE REVIEW

File: src/txorphanpool.h [hunk 1]
```diff
@@ -39,31 +39,34 @@ class CTxOrphanPool
 
     CTxOrphanPool();
 
-    //! Do we already have this orphan in the orphan pool
```
Comment (awemany) [inline src/txorphanpool.h]: Are you sure about this and the other changes removing the exclamation mark? It is the marker as it is used by `doxygen` for generating the code docs.

**Review by awemany — CHANGES_REQUESTED** [review]

**Comment by ptschip** [comment]
I don't really have any feeling one way or the other, was just trying
to be consistent with our comments everywhere else. I can change back.

File: src/txorphanpool.h (hunk 1 above)
Comment (ptschip) [reply src/txorphanpool.h]: ok done

**Comment by awemany** [comment]
Great @ Doxygen comments. Am I too pedantic if I also suggest that this should have a simple RPC test? :-)

**Comment by ptschip** [comment]
there is one , at the very top

**Comment by awemany** [comment]
Fair enough! I was thinking maybe to get something into the orphan pool and test that it appears in the RPC. (But I don't think that's worth blocking the PR)

**Pull request merged**